From f3f844d4cae2df48027b1f800d721409898576ca Mon Sep 17 00:00:00 2001
From: libin <bin.li@rock-chips.com>
Date: Wed, 18 Dec 2019 15:53:35 +0800
Subject: [PATCH 28/36] Android 10.0 HDR video adaptation:   1.Using struct
 hdr_static_metadata to get HDR information   2.Using struct
 hdr_output_metadata to enable HDMI HDR mode.

Change-Id: I4cd7ede1d3172e815f0e973856f193987e8d80c2
Signed-off-by: libin <bin.li@rock-chips.com>
---
 include/hardware/gralloc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/hardware/gralloc.h b/include/hardware/gralloc.h
index f4d716ac..4d2d7241 100755
--- a/include/hardware/gralloc.h
+++ b/include/hardware/gralloc.h
@@ -231,7 +231,7 @@ enum supported_hdmi_colorimetry {
 	COLOR_METRY_ITU_2020=9
 };
 
-struct hdr_static_metadata_1 {
+struct hdr_static_metadata {
        uint16_t eotf;
        uint16_t type;
        uint16_t display_primaries_x[3];
@@ -261,7 +261,7 @@ struct hdr_metadata_infoframe {
       __u16 max_fall;
 };
 
-struct hdr_static_metadata_2 {
+struct hdr_output_metadata {
     __u32 metadata_type;
     union {
         struct hdr_metadata_infoframe hdmi_metadata_type;
-- 
2.35.1

